### ISSUE TITLE
MessageBox: [A11Y] MessageBox content not read by screen reader after initial render on subsequent content update

### ISSUE THREAD

**Issue by fushizen** [SAP/ui5-webcomponents-react#7502]
### Describe the bug

When using a MessageBox dialog to guide the user through a process flow, the content is read by screen readers such as JAWS 2025 only the first time the dialog box is opened. Post user-interaction in the dialog box when the content is updated, the new content is not read by the screen reader. Video of behaviour is attached.

### Isolated Example

https://stackblitz.com/edit/github-3ajh2g-5vnemeew?file=src%2FApp.tsx

### Reproduction steps

1. Start JAWS
2. Go to the linked stackblitz
3. Open the dialog and press continue via mouse / keyboard controls
4. The content on first load is read by JAWS, but on dialog box content update after pressing continue, JAWS does not read any subsequent content

### Expected Behaviour

On an active and visible dialog box being updated with new information, the screen reader should read the new information being presented to the user

### Screenshots or Videos

https://github.com/user-attachments/assets/214ef6fa-8a9c-47b2-a0ec-7d6c8541dcea

### UI5 Web Components for React Version

2.10.0

### UI5 Web Components Version

2.10.0

### Browser

Edge, Chrome

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @fushizen 

this is the default behavior for all DOM elements. If you want screen readers to announce live updates you can use ARIA live regions, e.g. by leveraging the [Invisible Messaging ](https://sap.github.io/ui5-webcomponents/docs/advanced/accessibility/#invisible-messaging) feature of ui5-webcomponents. Here you can find a more detailed description of how invisible messaging is working: https://github.com/SAP/ui5-webcomponents/issues/11818#issuecomment-3022458349
